procmonAnalyze.js

```diff
@@ -474,17 +577,26 @@ function drawForeground() {
 
       entry.hiddenBySearch = !matchesSearch;
 
-      let startRelative = entry.start - minTime;
-      let endRelative = entry.end - minTime;
-      let startPixels = (startRelative + rendererTranslate) * rendererScale;
-      let endPixels = (endRelative + rendererTranslate) * rendererScale;
+      let startRelative = absTimeToRelTime(entry.start);
```

Comment:
Here is an example of the problem with `absTimeToRelTime` - we're subtracting `constrainedRelative` below to try to correct for the discrepancy, but we shouldn't need to do that. The renderer should be entirely in relative time, relative to `gState.minTime`, not `gOuterState.minTime`, because it shouldn't need to be aware of `gOuterState`. There are some leaky parts, like wanting to make sure that the timeline indicators and things like that use time relative to `gOuterState`, but I think that means we need another coordinate space, maybe "outer relative time" (`absTimeToOuterRelTime`). Is that making sense? (I could be wrong about this, but I think in the vast majority of cases things are simpler if we have relative time be relative to `gState.minTime`.)